apps/sensenet/package.json

```diff
@@ -64,7 +64,9 @@
     "react-router-dom": "^4.3.1",
     "source-map-loader": "^0.2.4",
     "style-loader": "^0.23.1",
+    "ts-config-webpack-plugin": "^1.3.1",
     "ts-jest": "^24.0.0",
+    "ts-node": "^8.0.2",
```

Comment:
ts-node is only necessary if the webpack config is a typescript file.

Reply:
👍